lib/spack/spack/spec.py

```diff
@@ -750,9 +754,126 @@ def __str__(self):
         return "{deps: %s}" % ', '.join(str(d) for d in sorted(self.values()))
 
 
+def _libs_default_handler(descriptor, spec, cls):
+    """Default handler when looking for 'libs' attribute. The default
+    tries to search for 'lib{spec.name}' recursively starting from
+    `spec.prefix`.
+
+    :param ForwardQueryToPackage descriptor: descriptor that triggered
```

Comment:
Hahaha, I switched to @alalazo's style too so that Sphinx could properly format our API docs. I figured I'd rather point people to the API docs than to the source code. Does Sphinx support automatic formatting of Google-style? I assume not.